lxd/instance/drivers/driver_lxc.go

```diff
@@ -3637,6 +3637,12 @@ func (d *lxc) Delete(force bool) error {
 
 // Delete deletes the instance without creating an operation lock.
 func (d *lxc) delete(force bool) error {
+	if !force {
```

Comment:
I experienced some locking issue here in case of broken concurrent snapshot creation so I think that if `force = true`, maybe we shoul not lock